redhat-cop/monitoring
# Add federate prometheus cluster

### What does this PR do?
The PR configures federation between prometheus_master and matches only job=prometheus.
For example if you have 3 prometheus_master nodes node_a, node_b, node_c
node_a targets federate endpoint on node_b, node_c
node_b targets federate endpoint on node_a, node_c
node_c targets federate endpoint on node_a, node_b


### How should this be tested?
Just run the automation on 2 prometheus_master and then check whatever metric starting with "prometheus_" prefix
The setup can be also checked in prometheus.yml in /home/centos

### Is there a relevant Issue open for this?
Provide a link to any open issues that describe the problem you are solving.
resolves #<number>

### Other Relevant info, PRs, etc.
Please provide link to other PRs that may be related (blocking, resolves, etc. etc.)

### People to notify
cc: @redhat-cop/monitoring
